pkg/server/v3/server_test.go

```diff
@@ -391,91 +392,91 @@ func TestFetch(t *testing.T) {
 	s := server.NewServer(context.Background(), config, cb)
 	out, err := s.FetchEndpoints(context.Background(), &discovery.DiscoveryRequest{Node: node})
 	assert.NotNil(t, out)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
Here and below, we can use `assert` instead of `require`. Again, if the linter insists, this is fine.